content/posts/2023-03-13-Hunt-for-and-craft-your-own-malicious-usb/index.mdx

```diff
@@ -0,0 +1,115 @@
+
+A malicious USB device can come in many different shapes and forms. Some are designed to look like ordinary USB drives, while others may be disguised as other devices such as keyboards, mice or books (as the title image). The goal is to lure unsuspecting victims into plugging the device into their computer or come close to an unlocked computer with a hidden USB, giving the attacker access to their system.
+
+One popular technique used by attackers is to create a device that looks like it contains personal information belonging to someone else. For example, in the TV series Mr. Robot, a character places a USB drive containing a fake resume in a parking lot. When an unsuspecting victim picks up the drive and plugs it into their computer, the attacker gains access to their system.
+
+But how do these devices work, and why are they so powerful? The answer lies in how computers trust USB devices. To use a computer normally, we need a keyboard to type on or a mouse to navigate. The computer trusts these devices to be legitimate and assumes that anyone with physical access to the computer is authorized to use them. Malicious USB devices take advantage of this trust to gain access to the system. By emulating a keyboard, the device can "type" commands into the computer, allowing the attacker to execute code or steal data. Because the device is seen as a legitimate input device, the computer doesn't question its actions, making it an effective tool for attackers.
+
+Although some vendors has tried to mitigate this by only allowing certain types of VIDs and PIDs, these are easily spoofable. [Take a look here](https://ducktoolkit.com/vidpid) how easy it may be to change VID and PID for Rubber Ducky. [Here](https://null-byte.wonderhowto.com/how-to/hack-macos-with-digispark-ducky-script-payloads-0198555/) is a bit more job for the Digispark.
+
+To protect yourself from malicious USB devices, it's important to follow a few simple rules. First and foremost, never plug in an untrusted device or one that you don't know where it came from. Always be wary of USB devices that you find in public places, as they may be designed to trick you into plugging them in.
+
+Secondly, make sure to sign out of your computer when you're done using it. This will help prevent unauthorized access by anyone who gains physical access to the device.
+
+# The basics of crafting a malicious USB drive
+
+There are several ways to get your hand on a malicious USB, the most popular is by far the Rubber Ducky by Hak5. The last few years there has been an increase in malicious cables, at least from my observations. The cable acts like a rubber ducky but looks like your normal USB charger cable. Another approach is to create one yourself using other popular microcontrollers. There are certain requirements for this to work but aiming for a Leonardo you are already there in terms of chipset that has this emulation feature.
```

Comment:
What is a leonardo?

Change the title? or deepen the insight into the crafting process?